src/Core/References/References.cs

```diff
@@ -41,25 +47,42 @@ public class References : IReferences
         /// The list of Packages that are automatically included for compilation. Namely:
         ///   * Microsoft.Quantum.Standard
         /// </summary>
-        public static readonly string[] BUILT_IN_PACKAGES =
-        {
-            "Microsoft.Quantum.Standard"
-        };
+        public readonly ImmutableList<string> BUILT_IN_PACKAGES =
+            ImmutableList.Create(
+                "Microsoft.Quantum.Standard"
+            );
```

Comment:
Since you're modifying this anyway, is it worth updating the name of this variable to CamelCase? Also perhaps something like `AutoLoadPackages` is a more accurate name, there's not really anything "built-in" about these packages, even less so with this change...

Reply:
Good point! This should now be fixed as of 9dc4953.

Reply:
Looks great, thanks for updating that!